docs/index.md

```diff
@@ -136,41 +233,46 @@ def api_update_user(request):
+  - **`response_status`**: A function that generates a response with a specified HTTP status code.
+  - Getting the singleton instance of YourCoreLib using YourCoreLibInstance.get(). This ensures that you're using the same instance of YourCoreLib throughout your application.
+
+- <b>Defining two API endpoint functions: api_update_user.</b>
+  - <i>**api_update_user (for POST requests)**:</i>
+    - **`Decorated with @require_POST`**: This decorator ensures that the endpoint only responds to POST requests.
+    - **`Decorated with @RequireLogin()`**: This decorator ensures that the user must be logged in to access this endpoint.
+    - **`Decorated with @HandleException()`**: This decorator handles any exceptions that occur within the endpoint function.
+    - Inside the function, your_core_lib.user.update() is called to update the user information using the data from the request body (request_body_dict(request)).
+    - Finally, it returns a response with HTTP status code NO_CONTENT using response_status(HTTPStatus.NO_CONTENT).
+  - <i>**api_update_user (for GET requests)**:</i>
+    - **`Decorated with @require_GET`**: This decorator ensures that the endpoint only responds to GET requests.
+    - **`Decorated with @RequireLogin()`**: This decorator ensures that the user must be logged in to access this endpoint.
+    - **`Decorated with @HandleException()`**: This decorator handles any exceptions that occur within the endpoint function.
+    - Inside the function, your_core_lib.user.update() is called to update the user information using the data from the request body (request_body_dict(request)).
+    - Finally, it returns a successful response using response_ok().
 
 ## The source
 
-[https://github.com/shay-te/core-lib](https://github.com/shay-te/core-lib)
+[https://github.com/shay-te/core-lib](https://github.com/shay-te/core-lib){:target="_blank"}
 
 ## Example project
 
-[https://github.com/shay-te/core-lib/examples](https://github.com/shay-te/core-lib/examples)
+[https://github.com/shay-te/core-lib/examples](https://github.com/shay-te/core-lib/examples){:target="_blank"}
 
 ## Contributing
 
-Please read [CONTRIBUTING.md](https://gist.github.com/PurpleBooth/b24679402957c63ec426) for details on our code of conduct and the process for submitting pull requests to us.
+Please read [CONTRIBUTING.md](https://gist.github.com/PurpleBooth/b24679402957c63ec426){:target="_blank"} for details on our code of conduct and the process for submitting pull requests to us.
 
 
 ## Authors
 
-**Shay Tessler**  - [GitHub](https://github.com/shay-te)
+**Shay Tessler**  - [GitHub](https://github.com/shay-te){:target="_blank"}
 
 
 ## License
 
-This project is licensed under the MIT - see the [LICENSE](https://github.com/shay-te/core-lib/blob/master/LICENSE) file for details.
-
+This project is licensed under the MIT - see the [LICENSE](https://github.com/shay-te/core-lib/blob/master/LICENSE){:target="_blank"} file for details.
```

Comment:
add a `next` button at the end of each page

Reply:
<img width="498" alt="image" src="https://github.com/shay-te/core-lib/assets/5166849/6ca888e8-ef62-4027-ab2d-98806446eae5">
fix the highlighting